onnx/onnxifi.h

```diff
@@ -1317,9 +1317,23 @@ ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI
  * Set locations for inputs and outputs of an ONNXIFI graph.
  *
  * The caller MUST ensure that the memory buffers specified for input and output
- * tensors remain accessible for the life-time of the ONNXIFI graph. The caller
- * can discard other data data in tensor descriptors, including shape, once the
- * function returns.
+ * tensors remain accessible until all in-flight graph executions which use
+ * specified buffer locations complete AND
+ * - Either a next call to onnxSetGraphIO specifies different buffer locations
+ * - Or the graph is deinitialized via onnxReleaseGraph
+ * The caller can invalidate other data data in tensor descriptors, including
+ * shape, once the function returns.
```

Comment:
What does this 'invalidate' mean? Will it cause next `onnxRunGraph` fail or should the backend detect any changes happened to the tensor descriptors across two `onnxRunGraph` calls?

Reply:
It means other data passed through tensor descriptors can be deallocated/changed/overwritten once `onnxSetGraphIO` returns.